app/addons/activetasks/components/tablebody.js

```diff
@@ -37,10 +37,11 @@ export default class ActiveTasksTableBody extends React.Component {
 
   noActiveTasks = () => {
     const type = this.getType();
-    let msg = <td  colSpan="6">No active {type} tasks.</td>;
+    const colSpan = 7 - this.props.hiddenColumns.length;
+    let msg = <td  colSpan={colSpan}>No active {type} tasks.</td>;
```

Comment:
Also a fix. When the new `node` column was added, the colspan should had been increment to 7.